### ISSUE TITLE
file-not-included is giving errors even when the file is included in the config

### ISSUE THREAD

**Issue by Frankyfrankly** [chakra-ui/eslint-plugin-panda#15]
Hi.
I was just following the installation guide for the basic React+Vite app, then installed and configured eslint-plugin-panda, but now the linter just keeps giving me errors on the `file-not-included` rule, saying that 
`The use of Panda CSS is not allowed in this file. Please check the specified `include` config  @pandacss/file-not-included`
Here are the steps to reproduce

1. Run the following set of commands:  
```sh
npm create vite@latest test-panda -- --template react-ts && cd .\test-panda\ && npm i && npm install -D @pandacss/dev && npx panda init --postcss && npm install -D @pandacss/eslint-plugin
```
2. Add `"prepare": "panda codegen",` to the `scripts` section of `package.json`
```json
 "scripts": {
    "prepare": "panda codegen",
    "dev": "vite",
    "build": "tsc && vite build",
    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
    "preview": "vite preview"
  },
```
3. Replace the entire content of `src/index.css` with the following line 
```css 
@layer reset, base, tokens, recipes, utilities;
```
4. Run `npm run dev`
5. Replace the content of `src/App.tsx` with the following code
```jsx
import { css } from '../styled-system/css';
 
function App() {
  return (
    <div className={css({ fontSize: "2xl", fontWeight: 'bold' })}>Hello 🐼!</div>
  )
}
 
export default App;
```
6. Add `"styled-system"` to the `include` section of `tsconfig.json`
7.  Enable the `recommended` rules in extends section of `.eslintrc` configuration file
```json
{
   "extends": ["plugin:@pandacss/recommended"]
}
```
8. (Optional) add `"styled-system"` to the `ignorePatterns` section of `.eslintrc`
9. Run `npm run lint` and experience the issue
`C:\test\test-panda\src\App.tsx
  1:1  error  The use of Panda CSS is not allowed in this file. Please check the specified `include` config  @pandacss/file-not-included`

Here is my setup:
![image](https://github.com/chakra-ui/eslint-plugin-panda/assets/3731823/47cb794d-dac2-4186-b9b5-33d497fd5e07)
![image](https://github.com/chakra-ui/eslint-plugin-panda/assets/3731823/14e72c2d-8f29-417d-92a5-77ed43f43b3e)

All I did was just following the [guide for installing the Panda CSS in React+Vite project](https://panda-css.com/docs/installation/vite), and[ the guide for installing `eslint-plugin-panda`](https://github.com/chakra-ui/eslint-plugin-panda)
Am I missing something?

**Comment by anubra266** [comment]
@Frankyfrankly You're looking for this step
https://panda-css.com/docs/installation/vite#configure-the-content

Ensure that the file you're styling in is covered by the glob in the `include` option

**Comment by anubra266** [comment]
@Frankyfrankly Do you still encounter this? I'm unable to reproduce

**Comment by Frankyfrankly** [comment]
@anubra266 It seems like indeed it has been fixed in one of the latest updates. Thanks!
